src/index.css

```diff
@@ -1,27 +1,83 @@
-body {
+:root {
+  --primary: #ffa500;
+  --secondary: #8c38ff;
+  --dark: #eee;
+  --light: #eee;
+}
+
+* {
   margin: 0;
-  font-family:
-    -apple-system,
-    BlinkMacSystemFont,
-    'Segoe UI',
-    'Roboto',
-    'Oxygen',
-    'Ubuntu',
-    'Cantarell',
-    'Fira Sans',
-    'Droid Sans',
-    'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    'Courier New',
-    monospace;
+  padding: 0;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+
+body {
+  background-color: var(--light);
+}
+
+.App {
+  display: flex;
+  min-height: 100vh;
+  align-items: center;
+  justify-content: center;
+  padding: 16px;
+}
+
+.calculator {
+  width: 100%;
+  max-width: 400px;
+  background-color: #fff;
+  border-radius: 16px;
+  overflow: hidden;
+  box-shadow: 0 2px 64px rgba(0, 0, 0, 0.2);
+}
+
+.display {
+  padding: 16px;
+  text-align: right;
+  background-color: #888;
+  color: #fff;
+  font-size: 24px;
+  font-weight: 300;
+}
+
+.display span {
+  font-size: 14px;
+  color: #888;
+}
+
+button {
+  appearance: none;
+  border: none;
+  outline: none;
+  color: #000;
+  font-size: 20px;
+  padding: 16px;
+  cursor: pointer;
+  transition: 0.4s;
+}
+
+button:hover {
+  opacity: 0.9;
+}
+
+.operators {
+  display: flex;
+}
+
+.operators button {
+  flex: 1 1 0%;
+  background-color: var(--primary);
+  font-weight: 700;
+}
+
+.digits {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.digits button {
+  flex: 1 1 33.333%;
+  max-width: 33.333%;
+  background-color: var(--dark);
 }
```

Comment:
- [ ] Kindly make sure your design sticks to the provided template. You are supposed to use the same colors and follow the provided styling(each number && symbol at each expected place/position).

| Template Styling | Your Styling  |
| ----------------------- | ------------------- |
| ![Screenshot from 2022-08-02 21-11-00](https://user-images.githubusercontent.com/83042742/182454579-6c42d0a7-3c7a-4acb-bfab-dc302c5353a4.png) | ![Screenshot from 2022-08-02 21-08-40](https://user-images.githubusercontent.com/83042742/182454151-5cfac8eb-f40b-4bf2-8127-4f0a27422204.png) |
 #

Reply:
@tresorsawasawa Thank you for the feedback, I have made the changes according to the styles and designs.